config/locales/en.yml

```diff
@@ -144,6 +144,8 @@ en:
     errors:
       duplicate_ssn: >
         An account already exists using the information you provided.
+      missing_finance: You must provide a financial account ID.
+      bad_dob: Your date of birth must be a valid date.
```

Comment:
The UI controls the format afaik. That's still in flux (see https://github.com/18F/identity-private/issues/701).